<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProTainer Studio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Protainer.png</normaloff>Protainer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="FileOpenButton">
      <property name="text">
       <string>OPEN A FILE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>View</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="SearchBar">
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>SEARCH FROM ITEM NAME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="ItemTable">
             <column>
              <property name="text">
               <string>Item Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Cost Price</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>MRP</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Profit%</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>UI MODE</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>829</width>
                 <height>260</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Name of Item</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ItemName"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Cost Price of Item</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ItemCost">
                  <property name="inputMethodHints">
                   <set>Qt::ImhNone</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Sell Price of Item</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ItemMRP">
                  <property name="inputMethodHints">
                   <set>Qt::ImhNone</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="AddItemUI">
                  <property name="text">
                   <string>Add Item From UI</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>JSON MODE</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QTextEdit" name="JSONCode"/>
            </item>
            <item>
             <widget class="QPushButton" name="AddItemJSON">
              <property name="text">
               <string>Add Item From JSON</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="FileNameLabel">
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>No File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
